### PR TITLE
Gs fixes3

### LINKED ISSUE #137: imos reconstruction does not work with excluded views
need to pass:
-EXCLUDELIST2 

From https://bio3d.colorado.edu/imod/doc/man/tilt.html

### LINKED ISSUE #147: verify the whole pipeline with excluded views

### PR DESCRIPTION
**Pull request by azazellochg**
To be merged after #148 
- fix generating ctf output; hide some params from manual ctf protocol; do not run autofit for manual protocol
- added xform file to ctfphaseflip to rotate astigmatism estimation and adjust for x-shifts for the image center
- fix excluded views parsing from com files
- rename "normalization" to "preprocess" protocols
- handle excluded views properly (closes #137 , closes #147)
- ctfplotter handles excluded views properly, output ctfs for such views have 0 values
- fix failed ts generation in fid.alignment protocol
- add number of threads to etomo protocol, so that ctfphaseflip and tilt can be run in parallel

### CODE REVIEW

File: imod/protocols/protocol_base.py [hunk 1]
```diff
@@ -417,75 +417,59 @@ def addCTFTomoSeriesToSetFromDefocusFile(self, inputTs, defocusFilePath, output)
 
         else:
             raise Exception(
-                "Defocus file flag do not supported. Only supported formats "
-                "corresponding to flags 0, 1, 4, 5, and 37.")
+                f"Defocus file flag {defocusFileFlag} is not supported. Only supported formats "
+                "correspond to flags 0, 1, 4, 5, and 37.")
 
+        excludedViews = inputTs.getExcludedViewsIndex()
         for index, _ in enumerate(inputTs):
```
Comment (pconesa) [inline imod/protocols/protocol_base.py:424]: This , although will work is a typical mistake (it was there before)

enumerating a set is a costly effort, and in this case is even worse cause we just want the index! and later we do the 

inputTS[index- +1]

So I'd suggest to either:

use the item in the enumerate or use set.getIds() and leave the inputTS[index- +1]

Comment (azazellochg) [reply imod/protocols/protocol_base.py:424]: got it. Found this in 11 files, will fix this now.

File: imod/protocols/protocol_tsNormalization.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ class ProtImodTSNormalization(ProtImodBase):
         https://bio3d.colorado.edu/imod/doc/man/newstack.html
     """
 
-    _label = 'Tilt-series normalization'
+    _label = 'Tilt-series preprocess'
```
Comment (pconesa) [inline imod/protocols/protocol_tsNormalization.py:45]: I prefer this name too!

**Review by pconesa — APPROVED** [review]
All good here

**Pull request merged**